source/projects/private-voice-assistant/voice_remote_esp32.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: "Setting up a $13 voice remote to control Home Assistant"
```

Comment:
```suggestion
title: "$13 voice remote to control Home Assistant"
```

Reply:
I'm using a system here. The -ing form is a visual cue telling readers that this is a task topic. A procedure that tells them step-by-step how to do something. So if they are looking for a procedure they can ignore anything that does not start with -ing. 

But I get that you are probably viewing this more from a marketing perspective.